common-tools/clas-tracking/src/main/java/org/jlab/clas/tracking/kalmanfilter/AStateVecs.java

```diff
@@ -414,7 +414,7 @@ public Helix getHelix(double xref, double yref) {
             if (Math.abs(phi0) > Math.PI) phi0 -= Math.signum(phi0) * 2 * Math.PI;
             double tanDip   = vec.tanL;
             double z0       = vec.z0 + vec.dz;
-            double omega    = -turningSign / R;
+            double omega    = turningSign * KFitter.polarity / R;  
```

Comment:
To make it more readable, this could become:

double omega = Math.signum(vec.kappa)/R;

or even better bypassing R

double omega = vec.kappa/vec.alpha;